index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="./css/styles.css">
+    </head>
+    <body>
+        <main>
+            <div class="column  logos-section">
+                <div class="row all-width">
+                    <a class="logo" href="#" aria-label="intuit"></a> 
+                    <br>   
+                </div class="row  all-width">
+                <div class="row all-width">
+                    <a href="#" class="mint-logo"></a>
+                    <a href="#" class="turbotax-logo"></a>
+                    <a href="#" class="quickbooks-logo"></a>
+                        
+                </div>
+            </div>
+            <div class="white-box">
+                <div>
+                    <h1>Create an Intuit account</h1>
+                    <p>One account for everything Intuit, including Mint. <a href="#">Learn more</a></p>
+                </div>
+                <div class="form-section">
+                    <form>
+                        <label>Email address</label><br>
+                        <input type="email"><br>
```

Comment:
#### `HINT`

:bulb: Consider associating the `<label>` with the `<input>` element, to do this you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's id. 
- Example 1: 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```
- Example 2: Alternatively, you can nest the `<input>` directly inside the `<label>` like this: 

``` html
  <label for="email">Email address <br>
       <input type="email" name="email" id="email">
  </label> 
```

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)